cuda_core/docs/source/interoperability.rst

```diff
@@ -0,0 +1,83 @@
+.. currentmodule:: cuda.core.experimental
+
+Interoperability
+================
+
+``cuda.core`` is designed to be interoperable with other Python GPU libraries. Below
+we cover a list of possible such scenarios.
+
+
+Current device/context
+----------------------
+
+The :meth:`Device.set_current` method ensures that the calling host thread has
+an active CUDA context set to current. This CUDA context can be seen and accessed
+by other GPU libraries without any code change. For libraries built on top of
+the CUDA runtime (``cudart``), this is as if ``cudaSetDevice`` is called.
```

Comment:
`cudart` is an ambiguous term in this context, so we should clarify (could mean `cuda_bindings.cudart),

Reply:
I removed the mention of `cudart` and added a hyperlink to the cudart docs in commit c06270c92b4a779314a029ac75c1645ab242b00c.